sdcm/mgmt.py

```diff
@@ -749,14 +747,23 @@ def __init__(self, manager_node):
     def _remoter_run(self, cmd):
         return self.manager_node.remoter.run(cmd)
 
-    def run(self, cmd, is_verify_errorless_result=False, parse_table_res=True, is_multiple_tables=False):
+    def run(self, cmd, is_verify_errorless_result=False, parse_table_res=True, is_multiple_tables=False,  # pylint: disable=too-many-arguments
+            replace_broken_unicode_values=True):
         LOGGER.debug("Issuing: 'sctool {}'".format(cmd))
         try:
+            # output_ascii_table=True,
+            # if output_ascii_table:
+            #     # setting SCTOOL_ASCII_TABLES so output of sctool will be in ascii
+            #     res = self._remoter_run(cmd="sudo bash -cxe 'SCTOOL_ASCII_TABLES=TRUE\nsctool {}'".format(cmd))
+            # else:
             res = self._remoter_run(cmd='sudo sctool {}'.format(cmd))
             LOGGER.debug("sctool output: %s", res.stdout)
         except (UnexpectedExit, Failure) as ex:
             raise ScyllaManagerError("Encountered an error on sctool command: {}: {}".format(cmd, ex))
 
+        if replace_broken_unicode_values:
+            res.stdout = res.stdout.replace('���', '│')
```

Comment:
this is the solution for this issue ?

Reply:
For now, because I don't want to be blocked by that.